### ISSUE TITLE
Membership Site - Access Control on Discussion Forums and Pages (similar to Courses)

### ISSUE THREAD

**Issue by tueolesen** [Simplero/Roadmap#142]
Courses in a Membership Site can be limited to only accessible for people who have purchaced specific Products.

Can Pages and Discussion Forum in a Membership site get the same access-control functionality?

It is possible to limit access for a discussion forum when using Spaces - so it's basically the same limitation, just for Membership Sites.

Essentially what i'm aiming at is to sell different tiers of access to the same Membership Site, so i want to limit certain areas of the Membership Site to customers who have purchased the high-end tier products giving them access.

<img width="708" alt="skaermbillede 2017-10-12 kl 13 48 07" src="https://user-images.githubusercontent.com/16045305/31516400-b75e0c52-af98-11e7-9d80-216638e353f0.png">

**Comment by cattwood108** [comment]
I just realized that this is essentially the same request I just made in the post: "Forum specific to course in Membership Site." 

So, can I add my vote for this functionality?

**Comment by stephenhutson** [comment]
I was about to suggest this exact thing. This would make my life so so so much more simple. Right now I'm having to auto-publish 2 version of the same page, then create automation that gives certain people access to one and not the other, then create triggers in the product section to trigger the appropriate automation. It's a small nightmare to deal with.

**Comment by tueolesen** [comment]
Any news on this one, Team Simplero?

**Comment by AnjaBusack** [comment]
Yes, this is exactly the same request I was making and thinking :)

**Comment by tueolesen** [comment]
Any news on this one? 

Forum content and forum support can be one of incentives for people to purchase a product - but right now we're giving it all away for free, to everyone who wants to read along.

I understand that the public access to forum content may be useful in some scenarios, to attract customers - but i would very much like being able to limit forum access to people who've purchased my products, to keep it more exclusive.

Even when no Forum links are shown to visitors on any of the site pages, forum posts are included in site search results and a forum.xml link is present in the HTML of the front page, letting search engines index all forum content.

**Comment by tueolesen** [comment]
Access control is SOLVED for pages.
Forums still lack access control - this is also created as a feature request here, to bring up visibility:
https://help.simplero.com/forum/posts/27241-implement-access-control-for-the-forum
